packages/widget.vite-plugin.unstable/src/build-plugin/extractBuildOutputs.ts

```diff
@@ -75,12 +75,14 @@ function visitNode(node: DefaultTreeAdapterTypes.Node): BuildOutput[] {
 function parseScriptNode(
   node: DefaultTreeAdapterTypes.Element,
 ): ScriptBuildOutput {
-  const srcAttribute = getAttribute(node, "src");
-  const typeAttribute = getAttribute(node, "type") as
-    | ScriptType
-    | undefined;
-  if (srcAttribute == null) {
-    throw new Error("Inline scripts are not supported");
+  assertEmptyNode(node);
+  assertAttributes(node, ["src", "type"]);
+  const srcAttribute = getAttribute(node, "src")!;
+  const typeAttribute = getAttribute(node, "type")!;
```

Comment:
We could also handle empty string or undefined for "classic" scripts:

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script/type#attribute_is_not_set_default_an_empty_string_or_a_javascript_mime_type